doc/nrf/app_dev/device_guides/nrf54l/non-volatile_storage.rst

```diff
@@ -0,0 +1,72 @@
+﻿.. _config_settings_zms:
+
+Non-volatile storage
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+For nRF54L Series, you should use the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>`.
+ZMS utilizes a flexible data management system that reduces write and erase cycles, extending the lifespan of flash memory.
```

Comment:
```suggestion
ZMS utilizes a flexible data management system that reduces write and erase cycles, extending the lifespan of non-volatile memory.
```

nRF54L has RRAM rather than flash.

Reply:
fixed